apps/web/src/services/auth/password/PasswordPage.vue

```diff
@@ -18,11 +18,16 @@
                                 {{ state.userInfo.userId }}
                             </span>
                         </p>
-                        <p v-else
+                        <p v-if="props.status === PASSWORD_STATUS.FIND"
                            class="help-text"
                         >
                             {{ $t('AUTH.PASSWORD.FIND.HELP_TEXT') }}
                         </p>
+                        <p v-else
+                           class="help-text"
+                        >
+                            {{ $t('AUTH.PASSWORD.INVALID_LINK.HELP_TEXT') }}
+                        </p>
```

Comment:
suggestion: What about to shorten this like below:

```suggestion
                        <p class="help-text">
                            {{ props.status === PASSWORD_STATUS.FIND ? $t('AUTH.PASSWORD.FIND.HELP_TEXT')  : $t('AUTH.PASSWORD.INVALID_LINK.HELP_TEXT') }}
                        </p>
```